akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardingQueries.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import akka.annotation.InternalApi
+
+object ShardingQueries {
```

Comment:
Needs `@InternalApi` marker and `private[sharding]` but might make sense to move into `ShardRegion` companion object to remove from public top-level scope of the `akka.cluster.sharding` package.

Reply:
I'd like to keep it separate from shard region to allow possible larger scope additions.
And originally this object was `@InternalApi private[sharding]` but I changed that due to a review suggestion here. ??

Reply:
Here: https://github.com/akka/akka/pull/27395#discussion_r307216236

Reply:
I suspect by "move that into the companion object" Johannes didn't mean "move the internal-api-markers into the `ShardingQueries` object", but "move the `ShardingQueries` object into the `ShardRegion` object"(?)

Reply:
Yes, sorry if that was confusing, but can also be done in a follow-up PR, I guess.